don't escape the banner output

http://bugs.gentoo.org/244222
https://bugzilla.mindrot.org/show_bug.cgi?id=1533

fix by Michał Górny <mgorny.3ehbo@mailnull.com>

--- sshconnect2.c
+++ sshconnect2.c
@@ -415,7 +415,7 @@ input_userauth_banner(int type, u_int32_t seq, void *ctxt)
 		if (len > 65536)
 			len = 65536;
 		msg = xmalloc(len * 4 + 1); /* max expansion from strnvis() */
-		strnvis(msg, raw, len * 4 + 1, VIS_SAFE|VIS_OCTAL);
+		strnvis(msg, raw, len * 4 + 1, VIS_SAFE|VIS_OCTAL|VIS_NOSLASH);
 		fprintf(stderr, "%s", msg);
 		xfree(msg);
 	}
